libs/langchain/langchain/retrievers/multi_vector.py

```diff
@@ -25,36 +26,26 @@ class MultiVectorRetriever(BaseRetriever):
     vectorstore: VectorStore
     """The underlying vectorstore to use to store small chunks
     and their embedding vectors"""
+    base_store: Optional[BaseStore[str, bytes]]
```

Comment:
should we call this `byte_store` instead?

If this was breaking in last release anyways, I think it's ok to rename (unless `base_store` in this is older than that)

Reply:
Sure, down for that

Reply:
Just confirming - this `base_store` param is brand new. Nobody would be using it?

Reply:
Can also use the `ByteStore` type instead of `BaseStore[str, bytes]`